<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winFunc</class>
 <widget class="QWidget" name="winFunc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>winFunc</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>窗口函数</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="rect_but">
        <property name="text">
         <string>矩形窗</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="han_but">
        <property name="text">
         <string>hanning窗</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ham_but">
        <property name="text">
         <string>hamming窗</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="kai_but">
        <property name="text">
         <string>Kaiser窗</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bmh_but">
        <property name="text">
         <string>BlackmanHarris窗</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bm_but">
        <property name="text">
         <string>Blackman窗</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCustomPlot" name="winFunc_wid" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
